.ci/linux/test.sh

```diff
@@ -23,16 +23,16 @@ do
 		continue
 	fi
 	dotnet add "$i" package XunitXml.TestLogger --version 2.0.0
-	dotnet add "$i" package coverlet.msbuild --version 2.5.1
 done
 
 # Run tests
-dotnet test -v n -r target -d target/diag.log \
+dotnet test ElasticApmAgent.sln -v n -r target -d target/diag.log \
```

Comment:
For some reason It did not work without the solution:

```
+ dotnet test -v n -r target -d target/diag.log '--logger:xunit;LogFilePath=TestResults.xml' '--collect:XPlat Code Coverage' --settings coverlet.runsettings /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:CoverletOutput=target/Coverage/ /p:Threshold=0 /p:ThresholdType=branch /p:ThresholdStat=total
MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains more than one project or solution file.

```